dmapiclient/search.py

```diff
@@ -119,6 +119,19 @@ def search_services(self, index, q=None, page=None, **filters):
 
         return response
 
+    def search_services_from_url(self, search_api_url, page=None):
+        scheme, netloc, path, params, query, fragment = urlparse(search_api_url)
+        query_params = parse_qsl(query)
+        if page:
+            query_params.append(('page', page))
+        query = urlencode(query_params)
+
+        paged_search_api_url = urlunparse((scheme, netloc, path, params, query, fragment))
```

Comment:
Is it worth forking urllib.parse and implementing this if we're going to start doing this a lot?

All this parse, append, unparse, etc. is really tedious.

Reply:
Sensible, but not now. :trollface: